src/components/Datatable/Pagination/Pagination.tsx

```diff
@@ -53,45 +51,35 @@ const Pagination: React.FC<PaginationProps> = ({
   canNextPage,
   pageIndex,
   isLoading,
+  numPageButtons = 8,
 }) => {
   const generatePages = (start, end) =>
```

Comment:
This method should go outside of the component, it's pure function no need to recreate on every render